runtime/parachains/src/inclusion.rs

```diff
@@ -289,6 +297,13 @@ impl<T: Config> Pallet<T> {
 					Error::<T>::ValidatorIndexOutOfBounds,
 				);
 
+				// If there is a bit set that shouldn't bet set, we ignore it.
```

Comment:
We should ignore the bit, not the entire bitfield. That was the previous behavior, but #3999 was meant to change it to allow those bitfields and ignore.

Reply:
We're supposed to have a check later on in the bitwise loop. But I guess it doesn't work right - it should probably just look into the `occupied_bitmask` instead.